.mergify.yml

```diff
@@ -15,8 +15,7 @@ pull_request_rules:
           - '-title~=.*CI.*SKIP'  # Cirrus-CI feature
           - '-title~=.*SKIP.*CI'  # "
           - 'status-success=success'  # defined in .cirrus.yml
-          # Temporarily disabled during repository bootstrapping
-          # - '#approved-reviews-by>=1'
+          - '#approved-reviews-by>=1'
```

Comment:
I have to assume that the `#` here, inside a string, is an actual part of the conditional. Is this under your control? Is there any possible way to remove the hash sign? It's super confusing. But I'll approve anyway.

Reply:
confirmed, here's the spot in docs: https://doc.mergify.io/conditions.html#conditions